common/python/cctrusted_base/tdx/qh.py

```diff
@@ -55,23 +94,88 @@ class TdxQuoteHeader(QuoteHeader):
     """
 
     def __init__(self, data: bytearray):
-        self._data = data
-        #TODO: parse raw data according to TD Quote format
-
-    def get_data(self) -> bytearray:
         """
-        Get raw data
+        Init function does 2 things:
+            1. Save raw data in its super property.
+            2. Parse the raw data and save each field as the properties of its "self".
         """
-        raise self._data
+        super().__init__(data)
+        v = memoryview(self.data)
+        self.ver = int.from_bytes(v[0:2].tobytes(), "little")
+        self.ak_type = AttestationKeyType(int.from_bytes(v[2:4].tobytes(), "little"))
+        self.tee_type = TeeType(int.from_bytes(v[4:8].tobytes(), "little"))
+        self.reserved_1 = v[8:10].tobytes()
+        self.reserved_2 = v[10:12].tobytes()
+        self.qe_vendor = v[12:28].tobytes()
+
+    def dump(self, fmt=Quote.QUOTE_OUT_FORMAT_RAW, indent=""):
+        """
+        Dump data. Default format is raw.
+        """
+        print(f'{indent}{type(self).__name__}:')
+        if fmt == Quote.QUOTE_OUT_FORMAT_HUMAN:
+            i = indent + "  "
+            print(f'{i}Header Version: {self.ver}')
```

Comment:
Please use LOG instead of print

Reply:
Changed. Please review the updated version. Thanks!